sql/telemetry/firefox_kpi_dashboard_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Firefox KPI dashboard
+description: Firefox KPI dashboard
```

Comment:
This is a placeholder for now, we'll probably want a more descriptive description